php/PhpJsonApiSamples/Default.html

```diff
@@ -31,41 +31,45 @@
 	<form id="form1" runat="server">
 	<table style="width:40%">
 		<tr><th>Select any option </th></tr>
-		<!--<tr> 
-			<td><a href="ClientSideEncryptionDemo.aspx">ClientSide Encryption </a> </td>
-		</tr>   -->
+		<tr> 
+                        <!-- Demo1 checks the client side validations and then submit the valid form with PayTrace client side library  -->
```

Comment:
So if i understand right the default.html page will be a list of examples for the API.
What do you think about changing the HTML comment to be instead HTML markup that displays on the default page?
Seems like if I was an integrator and I download the project, the default page would open in the browser showing me a list of examples, I would just click the link for the demo I am trying to research, so it might make it simpler if i didn't have to look at the markup comments to see what link i should pick.